helix-term/src/commands.rs

```diff
@@ -4808,14 +4808,18 @@ fn add_newline_impl(cx: &mut Context, open: Open) {
     apply_transaction(&transaction, doc, view);
 }
 
+enum IncrementDirection {
+    Increase = 1,
+    Decrease = -1,
```

Comment:
Since we don't end up using `increment_direction as isize`, the numbers here can be removed

Reply:
That was an accident. My bad.